docs/development/deploying-locally/index.md

```diff
@@ -18,62 +19,82 @@ $ source ~/.virtualenvs/reana/bin/activate
 
 ```console
 (reana) $ reana-dev git-fork -c ALL
-(reana) $ # read and run the printed eval
-(reana) $ GITHUB_USER=johndoe make clone
+(reana) $ eval "$(reana-dev git-fork -c ALL)"
+(reana) $ reana-dev git-clone -c ALL -u johndoe
 ```
 
-**3.** Start Minikube and prefetch base docker images:
+| Option                     | Description                                                                        | Default value             |
+|----------------------------|------------------------------------------------------------------------------------|---------------------------|
+| `--component` / `-c`       | Which components to clone (shortname, name, `.`, `CLUSTER`, `ALL`)                 | CLUSTER                   |
+| `--user` / `-u`            | GitHub user name.                                                                  | anonymous                 |
+
+**3.** Create a new REANA cluster:
 
 ```console
-(reana) $ make setup prefetch
+(reana) $ reana-dev cluster-create
 ```
 
+| Option                     | Description                                                                        | Default value             |
+|----------------------------|------------------------------------------------------------------------------------|---------------------------|
+| `--mount` / `-m`           | Which local directories to mount in the cluster (e.g. `-m /var/reana:/var/reana`)  | -                         |
+| `--mode`                   | Which mode to run REANA cluster (`releasehelm`, `releasepypi`,`latest`, `debug`)   | latest                    |
+
 **4.** Build all components:
 
 ```console
-(reana) $ make build
+(reana) $ reana-dev cluster-build
 ```
 
+| Option                     | Description                                                                        | Default value             |
+|----------------------------|------------------------------------------------------------------------------------|---------------------------|
+| `--build-arg` / `-b`       | Any build arguments (e.g. `-b COMPUTE_BACKENDS=kubernetes,htcondorcern,slurmcern`) | -                         |
+| `--exclude-components`     | Which components to exclude from build.                                            | -                         |
+| `--mode`                   | Which mode to run REANA cluster (`releasehelm`, `releasepypi`,`latest`, `debug`)   | latest                    |
+| `--no-cache`               | Do not use Docker image layer cache.                                               | false                     |
+
 **5.** Deploy REANA:
 
 ```console
-(reana) $ make deploy
+(reana) $ reana-dev cluster-deploy --admin-email john.doe@example.org --admin-password 123456
 ```
 
+| Option                     | Description                                                                        | Default value             |
```

Comment:
While it is cool to have the options listed, I'm pretty sure the list will get outdated soon after we shall add some new option to `reana-dev` in the future. 

So I'm not sure whether it is profitable to include these here... Developers read `reana-dev --help` if they want to know more?

It may be better to simply provide short realistic copy-pasteable `reana-dev` sequence of commands here without too much explanation. WDYT?

Reply:
I agree. Option tables can get outdated very quickly.

I will revert that change if there is no further discussion on this.